dev/com.ibm.ws.grpc.client/.classpath

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
```

Comment:
bnd requires java11 should be use that here as well?

Reply:
No other projects are doing this, so let's hold off for now.